docs/whats-new.asciidoc

```diff
@@ -4,172 +4,36 @@
+=== Logsdb index mode with detection rules and alerts
 
-Additional third-party response actions are available using Elastic's {security-guide}/third-party-actions.html#sentinelone-response-actions[SentinelOne] integration and connector:
+The {ref}/logs-data-stream.html[logsdb index mode] allows you to store log data more efficiently. If you're considering using it, refer to Using logsdb index mode with {elastic-sec} to learn how it can impact your rules and alerts.
 
-* Get processes
-* Terminate a process
+// link to be added when relevant PR is merged: {security-guide}/detections-logsdb-index-mode-impact.html[Using logsdb index mode with {elastic-sec}] 
 
 [float]
-=== {elastic-defend}'s automated response actions support all rule types
+=== Suppress alerts for EQL sequence rules
 
-You can now configure any detection rule type to perform {elastic-defend}'s {security-guide}/automated-response-actions.html[automated response actions].
-
-////
-Commenting out until docs are ready
+{security-guide}/alert-suppression.html[Alert suppression] now supports the EQL sequence rule type. You can use it to reduce the number of repeated or duplicate detection alerts generated from EQL sequence rules.
 
 [float]
-=== New rules for {elastic-defend}'s endpoint protection features
-
-New prebuilt rules tailored for each of {elastic-defend}'s endpoint protection features—malware, ransomware, memory threats, and malicious behavior—allow you to configure actions tailored for detection or prevention of each type.
+== Signature option available for macOS trusted applications conditions
 
-[role="screenshot"]
-image::whats-new/images/8.16/endpoint-protection-rules.png[Endpoint protection rules]
-////
+When adding a {security-guide}/trusted-apps-ov.html[trusted application] for macOS, you can now specify conditions based on the application's digital signer—previously only available on Windows.
 
 [float]
-== Cloud Security enhancements
-
-[float]
-=== Ingest third-party cloud security data
-
-You can now {security-guide}/ingest-third-party-cloud-security-data.html[ingest cloud security data] from several third-party sources—Falco, AWS Security Hub, and Wiz—into {elastic-sec}. The data appears on the **Alerts** and **Findings** pages, and in the user and host details flyouts.
-
-[role="screenshot"]
-image::whats-new/images/8.16/wiz-findings.png[Wiz data on the Findings page]
-
-[float]
-=== Simplify posture data collection with agentless Cloud Security Posture Management deployment
-
-Elastic's native {security-guide}/cspm.html[Cloud Security Posture Management (CSPM)] integration now supports agentless deployment, giving you an easier and more streamlined way to collect posture data from your cloud service providers.
+== Case action is generally available
 
+The {kibana-ref}/cases-action-type.html[Case action] feature, first introduced in 8.14, is moving from technical preview to general availability. Use Case action to automatically create cases from rules.
```

Comment:
Minor correction, based on the naming used in the [feature docs](https://www.elastic.co/guide/en/kibana/current/cases-action-type.html).
```suggestion
The {kibana-ref}/cases-action-type.html[Cases action] feature, first introduced in 8.14, is moving from technical preview to general availability. Use this action to automatically create cases from rules.
```